<ui version="4.0" >
 <class>TapStart_Base</class>
 <widget class="QWidget" name="TapStart_Base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>270</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>TapStart</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QPushButton" name="btnTap" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <family>Sans Serif</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text" >
      <string>Tap!</string>
     </property>
     <property name="default" >
      <bool>true</bool>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="lblTempoBuddy" >
     <property name="text" >
      <string>Tempo (bpm):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLabel" name="lblTempo" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>100</width>
       <height>25</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <family>Sans Serif</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text" >
      <string>XXX</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <property name="horizontalSpacing" >
      <number>6</number>
     </property>
     <property name="verticalSpacing" >
      <number>6</number>
     </property>
     <item row="0" column="1" >
      <widget class="QSpinBox" name="sBMeterTop" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="value" >
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QSpinBox" name="sBMeterBottom" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="value" >
        <number>4</number>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="0" column="0" >
      <widget class="QLabel" name="lblMeter" >
       <property name="text" >
        <string>Meter:</string>
       </property>
       <property name="buddy" >
        <cstring>sBMeterTop</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QGroupBox" name="jackBox" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="title" >
      <string>Start Jack Transport</string>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="cBBars" >
        <property name="text" >
         <string>Autostart after # bars:</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="sBBars" >
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>4</number>
        </property>
        <property name="value" >
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="btnMoreOsc" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>More OSC</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="3" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
    </layout>
   </item>
   <item row="5" column="0" colspan="3" >
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnTap</tabstop>
  <tabstop>cBBars</tabstop>
  <tabstop>sBBars</tabstop>
  <tabstop>sBMeterTop</tabstop>
  <tabstop>sBMeterBottom</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cBBars</sender>
   <signal>toggled(bool)</signal>
   <receiver>sBBars</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>157</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel" >
     <x>316</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
